File: backend/inventory-service/src/main/java/com/storelense/inventory/dto/BillLookupResponse.java
package com.storelense.inventory.dto;

import java.time.OffsetDateTime;
import java.util.List;
import java.util.UUID;

public record BillLookupResponse(
        UUID            id,
        String          billRef,
        UUID            storeId,
        OffsetDateTime  createdAt,
        List<BillItemDto> items,
        String          status,
        OffsetDateTime  gateCheckedAt
) {}

File: android-c66/app/src/main/java/com/storelense/c66/data/remote/dto/BillLookupResponse.kt
package com.storelense.c66.data.remote.dto

data class BillLookupItem(
    val ean:           String,
    val productName:   String?,
    val qty:           Int,
    val isRfidEnabled: Boolean? = null,
    val imageUrl:      String? = null
)

data class BillLookupResponse(
    val id:            String,
    val billRef:       String,
    val storeId:       String,
    val createdAt:     String,
    val items:         List<BillLookupItem>,
    val status:        String = "PENDING",
    val gateCheckedAt: String? = null
)
